sd_jwt/walk.py

```diff
@@ -1,39 +1,54 @@
 import json
 import logging
+from typing import Any, Callable, Dict, Optional, Tuple
 
 logger = logging.getLogger("sd_jwt")
 
 
-def by_structure(structure, obj, fn):
+def by_structure(structure: Dict, obj: Dict, fn: Callable[[str, str, Optional[Any]], Tuple[str, Any]], find_fn=lambda s, k: (k, s[k])):
```

Comment:
mmhhh ... probably nesting that lambda in find_fn is not for the people, can we have a simple None there a an if in the code that makes it more easy to be debugged?
````
if not find_fn:
    find_fn = lambda s, k: (k, s[k])
````

Reply:
That's right, will change.

Reply:
Fixed by making the 'default' function explicit.